clients/drcachesim/tests/memref_gen.h

```diff
@@ -84,7 +84,28 @@ gen_branch_encoded(memref_tid_t tid, addr_t pc, int encoding)
     memref.instr.encoding_is_new = true;
     return memref;
 }
+
+inline memref_t
+gen_syscall_encoded(memref_tid_t tid, addr_t pc, int encoding)
```

Comment:
Do we need to have the `encoding` parameter? It's the same in all invocations.

Reply:
Removed it.